### PR TITLE
fooyin: Add at 0.4.5

### PR DESCRIPTION
**Pull request by androidnisse**
**Summary**
- fooyin: Add at 0.4.5
- Resolves https://github.com/getsolus/packages/issues/2886

**Test Plan**
- Installed and checked it started correctly.

**Checklist**

- [X] Package was built and tested against unstable

### CODE REVIEW

File: packages/f/fooyin/package.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+release    : 1
+source     :
+    - https://github.com/fooyin/fooyin/archive/refs/tags/v0.4.5.tar.gz : d151807ccbff641caa791df45ee042ae508e45ec6e095af15d1f201d459e4ed8
+homepage   : https://www.fooyin.org/
+license    : GPL-3.0-or-later
+component  : multimedia.audio
+summary    : fooyin is a music player built around customisation.
+description: |
+    fooyin is a music player built around customisation.
+networking : yes
```
Comment (malfisya) [inline packages/f/fooyin/package.yml:12]: Why does it needs networking?

**Review by malfisya — CHANGES_REQUESTED** [review]
Let's use a more descriptive commit message.
```
package: Add at v1.0

```

Don't forget to also edit the github PR message

File: packages/f/fooyin/package.yml (hunk 1 above)
Comment (HarveyDevel) [reply packages/f/fooyin/package.yml:12]: > Why does it needs networking?

Networking can be disabled. Instead `kdsingleapplication-devel` needs to be added as a build dep. It couldn't find a system lib for it so it was pulling it as a git submodule.

```
-- Using 3rd-party KDSingleApplication
[1/9] Creating directories for 'kdsingleapplication-populate'
[1/9] Performing download step (git clone) for 'kdsingleapplication-populate'
Cloning into 'kdsingleapplication-src'...
fatal: unable to access 'https://www.github.com/KDAB/KDSingleApplication.git/': Could not resolve host: www.github.com

CMake Error at kdsingleapplication-subbuild/kdsingleapplication-populate-prefix/tmp/kdsingleapplication-populate-gitclone.cmake:39 (message):
  Failed to clone repository:
  'https://www.github.com/KDAB/KDSingleApplication.git'
```

Comment (malfisya) [reply packages/f/fooyin/package.yml:12]: Then, it probably better to use git (and the commit hash) as the source instead of tarball

Comment (HarveyDevel) [reply packages/f/fooyin/package.yml:12]: Just need to add the missing dep and git isn't needed at all.

**Review by malfisya — APPROVED** [review]
Looks good. Thank you!

**Pull request merged**